LibreNMS/Alert/Transport/Snmptrap.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/* Copyright (C) 2015 Daniel Preussker <f0o@devilcode.org>
+ ** This program is free software: you can redistribute it and/or modify
+ ** it under the terms of the GNU General Public License as published by
+ ** the Free Software Foundation, either version 3 of the License, or
+ ** (at your option) any later version.
+ **
+ ** This program is distributed in the hope that it will be useful,
+ ** but WITHOUT ANY WARRANTY; without even the implied warranty of
+ ** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ ** GNU General Public License for more details.
+ **
+ ** You should have received a copy of the GNU General Public License
+ ** along with this program.  If not, see <https://www.gnu.org/licenses/>. */
+
+/**
+ ** SNMP Trap Transport
+ **
+ ** @author Cloud Delivery Team <ttc-cloud-delivery@millicom.com>
+ ** @copyright 2022 Cloud Delivery Team, Tigo Technology Center
+ ** @license GPL
+ **/
+
+namespace LibreNMS\Alert\Transport;
+
+use LibreNMS\Alert\Transport;
+use LibreNMS\Config;
+
+class Snmptrap extends Transport
+{
+    /**
+     * @param  array  $obj
+     * @param  array  $opts
+     */
+    public function deliverAlert($obj, $opts)
+    {
+        $host = $this->config['snmptrap-destination-host'];
+        if (! empty($this->config['snmptrap-destination-port'])) {
+            $port = $this->config['snmptrap-destination-port'];
+        } else {
+            /* Default SNMP trap port */
+            $port = '162';
+        }
+        $transport = $this->config['snmptrap-transport'];
+        $trapdefinition = $this->config['snmptrap-definition'];
+        $pdu = $this->config['snmptrap-pdu'];
+        $community = $this->config['snmptrap-community'];
+        $binary = implode([Config::get('snmptrap')]);
```

Comment:
Why wrap in an array then implode?  Are you trying to cast to string?

Reply:
yup, any advice is welcomed :)

Reply:
`$binary = (string) Config::get('snmptrap');`


Additionally, it will be cast below in the cli call, so explicit cast is not needed.